.promu.yml

```diff
@@ -0,0 +1,17 @@
+go:
+    cgo: true
+repository:
+    path: github.com/oliver006/redis_exporter
+build:
+    flags: -a -tags 'netgo static_build'
+    ldflags: |
+        -s
+        -X {{repoPath}}/vendor/github.com/prometheus/common/version.Version={{.Version}}
+        -X {{repoPath}}/vendor/github.com/prometheus/common/version.Revision={{.Revision}}
+        -X {{repoPath}}/vendor/github.com/prometheus/common/version.Branch={{.Branch}}
+        -X {{repoPath}}/vendor/github.com/prometheus/common/version.BuildUser={{user}}@{{host}}
+        -X {{repoPath}}/vendor/github.com/prometheus/common/version.BuildDate={{date "20060102-15:04:05"}}
+tarball:
+    files:
+        - LICENSE
```

Comment:
Just tried `promu tarball` and it complains about missing NOTICE file so I think you can remove these two.